tests/foreman/rhai/test_rhai.py

```diff
@@ -103,14 +102,14 @@ def test_org_selection_for_rhai(self):
             self.assertIn("Organization Selection Required", result)
 
     @skip_if_not_set('clients')
-    def test_unregister_system_from_rhai(self):
+    def test_positive_unregister_client(self):
```

Comment:
s/test_positive_unregister_client/test_positive_unregister_client_to_rhat
i think 'rhai' word is very important here because we can register system to satellite/cdn/rhai. Good to add 'rhai' here.